docs/sql/general/constraints.rst

```diff
@@ -135,23 +130,11 @@ Multiple columns can be referenced::
 
 .. NOTE::
 
-   ``CHECK`` constraints cannot be added at "sub-column level" for object
-   sub-columns by referring simply to the sub-column name, e.g.::
-
-     CREATE TABLE t(o OBJECT AS (oi INTEGER CHECK (oi > 10)))
-
-   instead you need to provide the full path of that sub-column, e.g.::
-
-     CREATE TABLE t(o OBJECT AS (oi INTEGER CHECK (o['oi'] > 10)))
-
-   alternatively they can be added at "table level", e.g.::
-
-     CREATE TABLE t(o OBJECT AS (oi INTEGER), CHECK (o['oi'] > 100))
-
-   or at the "root level" of the object column, e.g.::
-
-     CREATE TABLE t(o OBJECT AS (oi INTEGER) CHECK (o['oi'] > 100))
```

Comment:
Removed this here because how to define table check constraints is already documented further above and this may give the impression that it's somehow special in regards to object columns, which it isn't.

Reply:
I agree, thx!